core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/handlers/main.yml

```
@@ -0,0 +1,21 @@
+---
+- name: restart postgres-exporter
+  service:
+    name: postgres-exporter
+    state: restarted
+  retries: 10
+  delay: 10
+
+- name: reload systemd daemons
+  systemd:
+    daemon_reload: yes
```

Let's keep it consistent. :+1:

```suggestion
    daemon_reload: true
```

---

Changed